code/datums/components/fantasy/_fantasy.dm

```diff
@@ -153,4 +150,4 @@
 		span = "<span class='danger'>"
 		effect_description = span_bold("mottled black glow")
 
-	location.visible_message("[span][originalName] is covered by a [effect_description] and then transforms into [parent]!</span>")
+	location.visible_message("[span]The [originalName] is covered by a [effect_description] and then transforms into [parent]!</span>")
```

Comment:
don't you want \The ?

Reply:
no

Reply:
also idk if it makes any difference to use \The or The